packages/components/src/tag/src/Tag.tsx

```diff
@@ -230,6 +231,9 @@ const StandaloneTag = forwardRef<FocusableElement, StandaloneTagProps>((props, r
     const { focusProps, isFocusVisible, isFocused } = useFocusRing({ within: true });
     const objectRef = useObjectRef(ref);
     const { linkProps } = useLink(props, objectRef);
+    // so we can use the same props as the RACTag
```

Comment:
@victortrinh2 i saw that the chromatic test started failing for the tag. You can revert everything in this file, and log a bug that you can't put a tooltip on a Tag that is not part of a TagGroup. I'll fix it in a separate PR when i get back